tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": ["es5", "dom"],
+    "strict": true,
+    "noEmit": true
+  },
+  "files": ["build/scroller.d.ts"]
+}
```

Comment:
@bespoyasov moved the questions about typescript in a separate comment

Which TypeScript versions we want to support (>=3.0, >=4.0, any)?

Reply:
Let's start with >=4.0 😺

Reply:
Then I should add in `package.json`:

```json
"peerDependencies": {
    "typescript": "^4.0.0"
  }
  ```

Reply:
I added it with next version of `npm` but it updated the whole `package-lock.json`

do you mind if I reran it and merge separately, but here instead I add only `peerDependency`?

PR with`lockfileVersion: 2` is  here – https://github.com/bespoyasov/scroller/pull/29

Reply:
Added typescript as peer dependency ✅

Reply:
Cool! Thanks a lot 😊